html/changelogs/BoxcarRacer41-bacon.yml

```diff
@@ -0,0 +1,5 @@
+author: "BoxcarRacer41"
+delete-after: True
+changes: 
+  - rscadd: "Added a new food, bacon. Bacon is made by processing raw meat cutlets."
+  - rscadd: "Added a new burger recipe, the Bacon Burger. It is made with one bun, one cheese wedge and three pieces of cooked bacon."
```

Comment:
Just so you know, we can now set the changelog directly in the description of the PR instead of having to create a file. When you create a new PR the template to make a changelog is already in the description.